chess.js

```diff
@@ -1541,14 +1615,46 @@ var Chess = function(fen) {
         }
       }
 
+      var to_hex = function(string) {
```

Comment:
Can you add a comment of too here to explain what you're doing with hex and URI encoding?  It's not immediately clear to me.  Thanks